conf/broker.yaml.template

```
@@ -1,3 +1,8 @@
 BROKER:
     # The path where your broker settings and inventory are located
     BROKER_DIRECTORY: "."
+    HOST_WORKFLOWS:
+        POWER_CYCLE: powercycle-vm
+        # POWER_OFF:
+        # POWER_ON:
+        # EXTEND:
```

I think this is correct, but can't check atm
```suggestion
        EXTEND: extend-vm
```

---

ah yeah, might as well add this one even though there aren't any tests/fixtures using it.